addons/binding/org.openhab.binding.chromecast/README.md

```diff
@@ -3,7 +3,7 @@
 The binding integrates Google Chromecast streaming devices.
 It not only acts as a typical binding, but also registers each Chromecast device as an audio sink that can be used for playback.
 
-In order play audio streams that are served from the runtime, the binding needs to know the url to access.
+In order play audio/video streams that are served from the runtime, the binding needs to know the url(Of your Openhab instalation) to access.
```

Comment:
The callback URL is currently only used for audio.

How about:

> When a Chromecast is used as audio sink, the Chromecast connects to the runtime to get the audio streams.
The binding sends the Chromecast URLs for getting the audio streams based on the Primary Address (Network Settings configuration) and the runtime HTTP port.
This default URL can be overridden with the Callback URL configuration parameter.
Configure a Callback URL for instance when using Apache/NGINX for HTTPS and authentication or when the Chromecast cannot connect using the Primary Address.